colorCodePair.cpp

```diff
@@ -0,0 +1,39 @@
+#include "colorCodePair.h"
+
+const char* MajorColorNames[] = { "White", "Red", "Black", "Yellow", "Violet" };
+const char* MinorColorNames[] = { "Blue", "Orange", "Green", "Brown", "Slate" };
+const int numberOfMajorColors = sizeof(MajorColorNames) / sizeof(MajorColorNames[0]);
+const int numberOfMinorColors = sizeof(MinorColorNames) / sizeof(MinorColorNames[0]);
+
+telecomColorCoder::MajorColor telecomColorCoder::ColorPair::fetchMajorColor() {
+    return _majorColor;
+}
+
+telecomColorCoder::MinorColor telecomColorCoder::ColorPair::fetchMinorColor() {
+    return _minorColor;
+}
+
+std::string telecomColorCoder::ColorPair::toColorPairString() {
+    std::string colorPairStr = MajorColorNames[_majorColor];
+    colorPairStr += " ";
+    colorPairStr += MinorColorNames[_minorColor];
+    return colorPairStr;
+}
+
+std::string telecomColorCoder::ColorPair::toColorPairReferenceManual(){
+    std::string colorPairStr = MajorColorNames[_majorColor];
+    colorPairStr += "-";
+    colorPairStr += MinorColorNames[_minorColor];
+    return colorPairStr;
+}
+//function returns the ColorPair object for givin pair number
```

Comment:
this comment does not add much value as the function name is self explanatory

Reply:
Removed the comment.